test/environment/rlimit.profile

```diff
@@ -1,5 +1,5 @@
-  rlimit-fsize 1024
```

Comment:
this one too

Reply:
Generic syntax checking for extra whitespaces is done here

https://github.com/netblue30/firejail/blob/f959fbb3a817dbf1452193b44efaebcc45e8cd83/test/profiles/profiles.sh#L18-L22

Hence this should only check for settings set via profile, not extra whitespace parsing

https://github.com/netblue30/firejail/blob/f959fbb3a817dbf1452193b44efaebcc45e8cd83/test/environment/environment.sh#L111-L112

Don't you think so?